launcher.js

```diff
@@ -0,0 +1,45 @@
+// A "launcher" to start the (Node version of the) router from the command line.
```

Comment:
Ok, this makes sense. I'm curious about your response to my comment above—just so I fully understand.